qiita_plugins/target_gene/tgp/util.py

```diff
@@ -0,0 +1,187 @@
+# -----------------------------------------------------------------------------
```

Comment:
Should this be qiita_plugins/util.py? It seems like a lot of the functionality here can be used by other plugins.